### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by Audrey-Ella-xo**
Hi Reviewer,

In this milestone, I did the following;
   - Added basic styling to the calculator.
   
I am looking forward to your favourable and swift feedback.
Warm regards

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 const Button = props => {
   const { name } = props;
   return (
-    <button type="button">
+    <button type="button" className="btn">
       {name}
```
Comment (Torres-ssf) [inline src/components/Button.js]: - [x] Button component should accept `color` and `wide` as incoming props.
Screenshot from 2021-01-13 12-48-25

- [x] Default `color` for the button should be orange.
Screenshot from 2021-01-13 12-49-46

- [x] `wide` is boolean, true means that button take 50% of space (used for `0` button)
Screenshot from 2021-01-13 12-50-39

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @Audrey-Ella-xo :hand: 
Since the version of your Calculator project, it's 1.0 and not 1.1, we need to apply the provided style to your project. We need to follow these requirements [here](https://microverse.pathwright.com/library/react-redux-v10/188148/path/step/107676524/) to style your project.

- [x] We need to use the provided style [here](https://github.com/microverseinc/project-react-calculator/blob/master/images/calculator.png)
Screenshot from 2021-01-13 12-41-52

- [x] The result should be provided in white and bold text.
Screenshot from 2021-01-13 12-43-26

- [x] Each row from the button panel should have 100px
Screenshot from 2021-01-13 12-45-00

Feel free to message me in Slack if you have any questions :smile:.
Happy coding :computer: 
**Sergio Torres**

File: src/components/Button.js [hunk 2]
```diff
@@ -2,14 +2,25 @@ import React from 'react';
...
+Button.defaultProps = {
+  color: '#F5913E',
+  wide: false,
+};
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+  color: PropTypes.string,
+  wide: PropTypes.bool,
+};
```
Comment (shloch) [inline src/components/Button.js:24]: Great job working on this component, you understood the TSE's request

**Review by shloch — APPROVED** [review]
## Hello @Audrey-Ella-xo 
Based on the corrections you made in relation to the former review, this milestone is given a thumbs up. The outcome is good.
calc

Happy coding

**Pull request merged**